Makefile

```diff
@@ -378,6 +378,12 @@ else
 	exit 1
 endif
 
+.PHONY: setup-prometheus-adapter
+setup-prometheus-adapter:  ## Setup prometheus adapter for VerticaAutoscaler
+	helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+	helm repo update
+	helm install prometheus-adapter prometheus-community/prometheus-adapter --namespace monitoring --create-namespace
```

Comment:
Make the release name and namespace a parameter. Default release name and namespace can be prometheus-adapter.

- There must be an undeploy command too
- The user must be able to set the prometheus url, prometheus port , the number of replicas.